src/test/java/org/jabref/logic/citationkeypattern/BracketedPatternTest.java

```diff
@@ -380,4 +385,26 @@ void expandBracketsWithTestCasesFromRegExpBasedFileFinder() {
                 "Eric von Hippel and Georg von Krogh have published Open Source Software and the \"Private Collective\" Innovation Model: Issues for Organization Science in Organization Science.",
                 BracketedPattern.expandBrackets("[author] have published [title] in [journal].", ',', entry, database));
     }
+
+    @ParameterizedTest
+    @MethodSource("provideArgumentsForFallback")
+    void expandBracketsWithFallback(String expandResult, String pattern) {
+        BibEntry bibEntry = new BibEntry()
+                .withField(StandardField.YEAR, "2021").withField(StandardField.EPRINT, "2105.02891");
+        BracketedPattern bracketedPattern = new BracketedPattern(pattern);
+
+        assertEquals(expandResult, bracketedPattern.expand(bibEntry));
+    }
+
+    private static Stream<Arguments> provideArgumentsForFallback() {
+        return Stream.of(
+                Arguments.of("auth", "[title:(auth)]"),
+                Arguments.of("auth2021", "[title:(auth[YEAR])]"),
+                Arguments.arguments("not2021", "[title:(not[YEAR])]"),
+                Arguments.arguments("", "[title:([YEAR)]"),
+                Arguments.arguments(")]", "[title:(YEAR])]"),
+                Arguments.arguments("2105.02891", "[title:([EPRINT:([YEAR])])]"),
+                Arguments.arguments("2021", "[title:([auth:([YEAR])])]")
```

Comment:
Why are you using both `arguments` and `of`? Isn't `arguments` [meant for static import](https://junit.org/junit5/docs/5.4.0/api/org/junit/jupiter/params/provider/Arguments.html#arguments(java.lang.Object...))?

Reply:
Sorry, my mistake.I have changed them to `Arguments.of`.